button-examples/Web/web-button-example.md

```diff
@@ -19,43 +21,52 @@ Before you can use a button, you will need to install the following:
 * [Add a theme](#add-a-theme)
 * [Import JavaScript effects](import-javascript-effects)
 
-### Install `mcd-button` or `mdc-icon-button`
+### Install
 Install the `mdc-button` component before including it in your source.
 
+TODO: Add subsection for CDN users (Who do not use any bundler).
+
 **`mdc-button`**
 ```bash
-npm install @material/buttons
+npm install @material/button
 ```
 
+If you would like to use Icons as buttons:
+
 **`mdc-icon-button`**
 ```bash
 npm install @material/icon-button
 ```
 
-### Add a theme
-Both `mdc-button` and `mdc-icon-button` work with themes (styles). Import a style into your `*.scss*` stylesheet to apply it to your website, including buttons or toggle icon buttons:
+### Add styles
+Import component styles in your  `*.scss` stylesheet to apply it to your website, including buttons or toggle icon buttons:
+
+TODO: Link reference about how to setup Sass.
 
 **mdc-button**
-```css
+```scss
 @import "@material/button/mdc-button";
 ```
 
 **mdc-icon-button**
-```css
+```scss
 @import "@material/icon-button/mdc-icon-button";
 ```
 
-### Import JavaScript effects
-You can also add a JavaScript ripple effect (see [MDC Ripple](https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple)) to your buttons by importing and then instantiating `MCDRipple` in your `*.js` file. See the page on importing the [JavaScript component](https://github.com/material-components/material-components-web/blob/master/docs/importing-js.md) for more information on importing JavaScript.
+### JavaScript
+Optionally add ripple effect (see [MDC Ripple](https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple)) to your buttons by importing and then instantiating `MDCRipple` in your `*.js` file. See the page on importing the [JavaScript component](https://github.com/material-components/material-components-web/blob/master/docs/importing-js.md) for more information on importing JavaScript.
+
+TODO: Link to reference how to bundle JS.
```

Comment:
Do you have a link to the JS bundle instructions?

Reply:
Sure. https://github.com/material-components/material-components-web/blob/master/docs/getting-started.md#quick-start-cdn